lib/readyset/railtie.rb

```diff
@@ -13,5 +13,20 @@ class Railtie < Rails::Railtie
         ActiveRecord::Relation.prepend(Readyset::RelationExtension)
       end
     end
+
+    initializer 'readyset.query_annotator' do |app|
+      ActiveSupport.on_load(:after_intialization) do
+        if Rails.configuration.active_record.query_log_tags_enabled
+          Rails.configuration.active_record.query_log_tags ||= []
+          Rails.configuration.active_record.query_log_tags << {
+            routed_to_readyset?: ->(context) do
+              Readyset::QueryAnnotator.routing_to_readyset?
+            end,
+          }
+        else
+          Rails.logger.warn 'Query log tags are either disabled or unavailable.'
```

Comment:
I like this warn idea! I think we could add a little more context -- maybe something like this:
> Query log tags are currently disabled. The readyset gem uses these tags to display information in the logs about whether a query was routed to ReadySet. It is highly recommended that you enable query log tags by setting `Rails.configuration.active_record.query_log_tags_enabled` to `true` so you can verify that queries are being routed to ReadySet as expected.

Reply:
That's an excellent consideration. I'll be sure to add that.

Reply:
Adjusted it! It's now operating under a `Rails.env.development?` condition.